rcl/src/rcl/localhost.c

```diff
@@ -0,0 +1,30 @@
+// Copyright 2019 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <rcl/localhost.h>
+
+#include "rcutils/get_env.h"
+
+#include <stdlib.h>
+#include <string.h>
+
+const char * RCL_LOCALHOST_ENV_VAR = "ROS_LOCALHOST_ONLY";
```

Comment:
likewise, consider:
```suggestion
const char * const RCL_LOCALHOST_ENV_VAR = "ROS_LOCALHOST_ONLY";
```

Reply:
and also could be static value, since the scope is only localhost.c file ?